src/compas_view2/objects/meshobject.py

```diff
@@ -11,51 +11,58 @@ class MeshObject(BufferObject):
     ----------
     data : :class: `compas.datastructures.Mesh`
         Mesh for the viewer
+    vertices : list
+        Subset of vertices to be displayed
+    edges : list
+        Subset of edges to be displayed
+    faces : list
+        Subset of faces to be displayed
+    hide_coplanaredges : bool
+        True to hide the coplanar edges
+
+    Parameters[kwargs]
```

Comment:
@brgcode For classes that have inherited parents inputs, we can add a section of `Parameters[kwargs]` (not sure if it's best name tho), indicating these are the additional parameters that can be passed as key word arguments.

Reply:
you cannot choose the sections freely. if we want to add a new section this needs to be integrated in the theme. but there is already "Additional Parameters"

Reply:
ah ok then let's just use additional parameters